modules/afsocket/afsocket-dest.c

```diff
@@ -407,7 +451,9 @@ afsocket_dd_save_connection(AFSocketDestDriver *self)
 
   if (self->connections_kept_alive_accross_reloads)
     {
-      cfg_persist_config_add(cfg, afsocket_dd_format_persist_name(self, FALSE), self->writer, (GDestroyNotify) log_pipe_unref, FALSE);
+      GlobalConfig *cfg = log_pipe_get_config(&self->super.super.super);
+      ReloadStoreItem *item = _reload_store_item_new(self);
```

Comment:
strange situation, but well... why not: a review note for me by myself
Why I redefined the cfg variable?
(I had a separate function, but finally I removed because that was unnecessary, so copy-pasted the body here and forgot to remove the cfg.)
I'll update my PR.